src/rod/builder/primitives.py

```diff
@@ -54,3 +59,28 @@ def _geometry(self) -> rod.Geometry:
         return rod.Geometry(
             cylinder=rod.Cylinder(radius=self.radius, length=self.length)
         )
+
+
+@dataclasses.dataclass
+class MeshBuilder(PrimitiveBuilder):
+    mesh_path: Union[str, pathlib.Path]
+    scale: ArrayLike
+
+    def __post_init__(self) -> None:
+        self.mesh: trimesh.base.Trimesh = trimesh.load(
+            str(self.mesh_path), force="mesh"
+        )
+        assert self.scale.shape == (
+            3,
+        ), f"Scale must be a 3D vector, got {self.scale.shape}"
+
+    def _inertia(self) -> rod.Inertia:
+        inertia = self.mesh.moment_inertia
+        return rod.Inertia(
+            ixx=inertia[0, 0],
+            iyy=inertia[1, 1],
+            izz=inertia[2, 2],
+        )
+
+    def _geometry(self) -> rod.Geometry:
+        return rod.Geometry(mesh=rod.Mesh(uri=str(self.mesh_path), scale=self.scale))
```

Comment:
I'm not super familiar with meshes since I never had the need to directly create nor manipulate them. My intuition is that the mesh definition could also include a roto-translation that is usually mapped in the [`pose`](https://github.com/lorycontixd/rod/blob/91f2332edefbb360cae9bd466848f7e392d5652c/src/rod/sdf/visual.py#L18) attribute of the `rod.Visual` that includes the `rod.Geometry` of a mesh. Does a generic `trimesh.base.Trimesh` object provide such transform?

I also cc @traversaro that has more experience on the topic.

Reply:
I am not sure about this, it seems that the rest of `rod.Geometry` do not include any pose information, why mesh should be different?

Reply:
> If we can assume in the most generic way that a generic `Trimesh` object is always defined without the need of specifying additional transforms, the current status works perfectly fine. I was just wondering if this is always true.

Going trough `trimesh.base.Trimesh` that seems the case.